vllm/logging_utils/__init__.py

```diff
@@ -0,0 +1,5 @@
+from vllm.logging_utils.formatter import NewLineFormatter
+
+__all__ = [
+    "NewLineFormatter",
+]
```

Comment:
It would be ideal if this file was a `git mv` as well, like `formatter.py`

Reply:
the content changes from  `vllm.logging` to `vllm.logging_utils`

Reply:
only 1 line changes, no

Reply:
actually, it's correct in git, and just not displayed like that in github's diff viewer.